tests/testthat/test-hooks.R

```diff
@@ -118,7 +118,7 @@ run_test("style-files",
 run_test("style-files",
   file_name = "style-files-cmd",
   suffix = "-success.R",
-  std_err = "scope must be one",
+  std_err = ifelse(packageVersion("styler") < package_version("1.10.3"), "scope must be one", "`scope` must be one"),
```

Comment:
```suggestion
  std_err = "scope",
```

Maybe it would be best to change this to only scope. Maybe rlang could change its error messages from `rlang::arg_match()` and this would break precommit again.

The tidyverse team advocates not to test error messages from other packages. I guess this applies to packages from the same maintainer?

Reply:
Maybe yeah. So should we just test that there is an error without checking the exact message? I think to be honest, since the next release is pressing, I'd probably do that now. In the mid to long term, I'd be interested in refactoring the testing infra. Would you be interested in supporting this? I created an issue and outlined how we could start that: https://github.com/lorenzwalthert/precommit/issues/552

Reply:
I will see, but I was under the expression this was a regex, similar to when you use `expect_error()`. my proposal was to simplify that regex to "scope" to avoid future change in the error message from styler  affecting precommit.

Reply:
It is not a regex, we check if the string is a sub string of the error message, since `fixed` is `TRUE`. But regardless, we checking for `scope` would still work. I just went for the empty sting to eliminate any dependency on the error message. Because testing for `scope` still has a dependency, albeit a very minimal one.